src/custom/pages/Profile/LockedGnoVesting/claimData/index.ts

```diff
@@ -0,0 +1,60 @@
+import mainnetIndex from './mainnet.json'
+import rinkebyIndex from './rinkeby.json'
+import gnosisChainIndex from './gnosisChain.json'
+import { SupportedChainId } from '@src/custom/constants/chains'
```

Comment:
please change all paths prepended with `@src/custom/` to have it removed
e.g 
`@src/custom/abis/types` => `abis/types`
same if `./custom/*` => `*.ts` or equivalent. 
we have pathing aliases set up in tsconfig